templates/kubernetes/index.html

```diff
@@ -204,9 +204,10 @@ <h2>Kubernetes with Canonical support</h2>
           </li>
           <li class="p-list__item is-ticked">Fine-grained service placement</li>
         </ul>
-        <p>
-          <a href="/kubernetes/install#multi-node">Install Charmed Kubernetes&nbsp;&rsaquo;</a>
-        </p>
+        <ul class="p-inline-list">
```

Comment:
I think its just `p-list`
```suggestion
        <ul class="p-list">
```

Reply:
It's `p-list` for regular lists but `p-inline-list` for [inline lists](https://vanillaframework.io/docs/patterns/lists#inline) 🙂